src/unit/format.js

```diff
@@ -0,0 +1,49 @@
+define([
+	"../common/format-message"
+], function( formatMessage ) {
+
+/**
+ * format( value, unit, pluralGenerator )
+ *
+ * @value [Number]
+ *
+ * @unitProperies [Object]: localized unit data from cldr.
+ *
+ * @pluralGenerator [Object]: A pluralGenerator from Globalize.pluralGenerator.
+ *
+ * TODO pass along numberFormatter
```

Comment:
Oh! So, here it is. I am ok to tracked this as a separate issue. I will let @alunny answer if he is comfortable including this in this PR or to leave it to a subsequent one.